### PR TITLE
[13.0][MIG] stock_picking_procure_method: Migration to 13.0

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT27223

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
:+1: tested

File: stock_picking_procure_method/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2018 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Picking Procure Method",
+    "summary": "Allows to force the procurement method from the picking",
+    "version": "13.0.1.0.0",
+    "category": "Warehouse",
+    "author": "Tecnativa," "Odoo Community Association (OCA)",
```
Comment (victoralmau) [inline stock_picking_procure_method/__manifest__.py]:
```suggestion
    "author": "Tecnativa, Odoo Community Association (OCA)",
```

**Review by victoralmau — APPROVED** [review]

**Comment by ernestotejeda** [comment]
@victoralmau changes done

**Pull request merged**